python3/WcaOauth.py

```diff
@@ -0,0 +1,44 @@
+#!env/bin/python
```

Comment:
nit: Does this work for you? I think this should be an absolute path (starting with `/`)

Reply:
I just noticed that Mac OS is ignoring it if it can't be found or is not compatible. I'll fix it to use whichever binary it finds in `$PATH` though.